### ISSUE TITLE
Build failure: handbrake

### ISSUE THREAD

**Issue by Thatoo** [NixOS/nixpkgs#353778]
### Steps To Reproduce

1. build handbrake

### Build log

```
building the system configuration...
error: builder for '/nix/store/8fsbv1wpjv51izb4w01n0dw1n4hnqx2y-ffmpeg-full-7.0.2.drv' failed with exit code 2;
       last 25 log lines:
       > CC   libavcodec/libvo-amrwbenc.o
       > CC libavcodec/libvorbisdec.o
       > CC   libavcodec/libvorbisenc.o
       > CC   libavcodec/libvpxdec.o
       > CC      libavcodec/libvpxenc.o
       > CC      libavcodec/libwebpenc.o
       > CC     libavcodec/libwebpenc_animencoder.o
       > CC libavcodec/libwebpenc_common.o
       > CC      libavcodec/libx264.o
       > CC        libavcodec/libx265.o
       > CC        libavcodec/libxavs.o
       > CC        libavcodec/libxevd.o
       > CC        libavcodec/libxeve.o
       > CC        libavcodec/libxvid.o
       > CC        libavcodec/libzvbi-teletextdec.o
       > libavcodec/libxeve.c: In function 'get_conf':
       > libavcodec/libxeve.c:199:27: error: incompatible types when assigning to type 'XEVE_RATIONAL' {aka 'struct _XEVE_RATIONAL'} from type 'long int'
       >   199 |         cdsc->param.fps = lrintf(av_q2d(avctx->framerate));
       >       |                           ^~~~~~
       > libavcodec/libxeve.c: In function 'libxeve_encode':
       > libavcodec/libxeve.c:484:40: error: incompatible types when assigning to type 'int' from type 'XEVE_RATIONAL' {aka 'struct _XEVE_RATIONAL'}
       >   484 |                 avpkt->time_base.den = xectx->cdsc.param.fps;
       >       |                                        ^~~~~
       > make: *** [ffbuild/common.mak:81: libavcodec/libxeve.o] Error 1
       > make: *** Waiting for unfinished jobs....
       For full logs, run 'nix log /nix/store/8fsbv1wpjv51izb4w01n0dw1n4hnqx2y-ffmpeg-full-7.0.2.drv'.
error: 1 dependencies of derivation '/nix/store/z24d33jbrw5nrznzj5mql2y4byvz3rdz-handbrake-1.8.2.drv' failed to build
error: 1 dependencies of derivation '/nix/store/361m5r4q7g3yrz16dshkxza64lfqj7ia-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/mzkicrd1d5fnnndd5r8ci8ymyg1q9vdp-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/syiq3aqbc5if2j8dncv724jr02z9gkli-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/zlpwzivgph7vmpnsqcdspa795yykrrps-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/v50zndgamfy886vv31w8c9mq75759rdq-nixos-system-ThinkPadT470-24.11.20241102.7ffd9ae.drv' failed to build
```

### Additional context

NixOS unstable (24.11)

### Notify maintainers

@Anton-Latukha @wmertens 

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.59, NixOS, 24.11 (Vicuna), 24.11.20241102.7ffd9ae`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos-24.05"`
 - nixpkgs: `/nix/store/zq2axpgzd5kykk1v446rkffj3bxa2m2h-source`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by itepastra** [comment]
should be fixed when #353198 hits `nixos-unstable`

**Comment by Atemu** [comment]
Duplicate of https://github.com/NixOS/nixpkgs/issues/353072
